assets/javascripts/vendor/markdown-it-footnote.js

```diff
@@ -364,5 +365,5 @@ module.exports = function footnote_plugin(md) {
   md.core.ruler.after('inline', 'footnote_tail', footnote_tail);
 };
 
-},{}]},{},[1])(1)
+},{}]},{},[1])(1);
```

Comment:
👀 was this prettier, before .prettierignore got updated? 🙂

Reply:
Must've -- I think my editor linter caught it. Will fix.